src/modules/countLike.test.js

```diff
@@ -0,0 +1,17 @@
+import countLikes from './countLikes.js';
+
+describe('countLikes', () => {
+  test('adds the number of movies to the count', () => {
+    // Arrange
+    const countMovies = {
+      textContent: 'Number of Movies',
+    };
+    const moviesLength = 5;
+
+    // Act
+    countLikes(countMovies, moviesLength);
+
+    // Assert
+    expect(countMovies.textContent).toBe('Number of Movies (5)');
+  });
+});
```

Comment:
- [ ] Nice job for adding a test case for _Like_ counter, however, we intend that you provide test cases for other counters such as the _Home_ counter (_Counter for a number of movies on the home screen_) and the _Comment_ counter as well.
 - [ ]  Please also make sure to create tests for **all** possible edge cases of the counter functions, for instance, when the counter value is 0;

Reply:
> * [ ]  Nice job for adding a test case for _Like_ counter, however, we intend that you provide test cases for other counters such as the _Home_ counter (_Counter for a number of movies on the home screen_) and the _Comment_ counter as well.
> * [ ]   Please also make sure to create tests for **all** possible edge cases of the counter functions, for instance, when the counter value is 0;

- [ ] I see you have removed the whole test in its entirety, kindly know that it is a core requirement that can't be overlooked, From our chat in Slack, I understand that you are quite concerned about the time but take your time to work sufficiently and understand it. You can always write to Students' Success to even help you if you work on this project over the weekend and then pass it on Monday, Students' Success can help you, group to the new week, I have experienced this before. They are there for our progress. I hope you understand. Also, remember to fix the whole recommendation of the previous reviewer.